src/sentry/api/endpoints/internal/integration_proxy.py

```diff
@@ -183,6 +185,15 @@ def http_method_not_allowed(self, request):
         self.log_extra["host"] = request.headers.get("Host")
 
         if not self._should_operate(request):
+            if self.integration.provider == "slack":
```

Comment:
looks like integration is initialized in some other func via `self.integration = self.org_integration.integration`